src/Aspire.Hosting/Dashboard/proto/resource_service.proto

```diff
@@ -140,11 +129,6 @@ message Resource {
     optional google.protobuf.Timestamp created_at = 6;
     repeated EnvironmentVariable environment = 7;
 
-    // Deprecated and replaced with urls
-    optional int32 expected_endpoints_count = 8 [deprecated = true];
-    repeated Endpoint endpoints = 9 [deprecated = true];
-    repeated Service services = 10 [deprecated = true];
```

Comment:
Reserve the deleted field numbers. See https://protobuf.dev/programming-guides/proto3/#reserved-field-numbers